### PR TITLE
Add an ability to read and write translations

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Depends on  | https://github.com/yiisoft/translator/pull/94

### CODE REVIEW

**Review by rustamwin — APPROVED** [review]

File: src/Controller/InspectController.php [hunk 1]
```diff
@@ -43,6 +45,40 @@ public function config(ContainerInterface $container, ServerRequestInterface $re
...
+                'Make sure that "$params[\'locale\'][\'locales\']" contains all available locales.'
+            );
+        }
+        $messages = [];
+        foreach ($categorySources as $categorySource) {
+            $messages[$categorySource->getName()] = [
+                'messages' => [],
+            ];
+
+            try {
```
Comment (vjik) [inline src/Controller/InspectController.php:70]: Why need `try ... catch` here?

Comment (xepozz) [reply src/Controller/InspectController.php:66]: Some readers may fail while reading. It's unnecessary to have the fault in the debugger, but necessary to have at least something.

Comment (vjik) [reply src/Controller/InspectController.php:66]: Suggest for categories/locales with error return null or false value, that mean error. It's allow immediately see problem in debug panel.

Comment (xepozz) [reply src/Controller/InspectController.php:66]: I'll do it later

**Review by vjik — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
         "yiisoft/di": "^1.0",
         "yiisoft/http": "^1.2",
         "yiisoft/router": "^2.0",
+        "yiisoft/translator": "^2.1",
```
Comment (vjik) [inline composer.json:34]:
```suggestion
        "yiisoft/translator": "^2.2",
```

**Pull request merged**